routes/web.php

```diff
@@ -18,23 +18,23 @@
 
 // Socialite
 Route::prefix('auth')->name('socialite.')->group(function () {
-    Route::post('{provider}/redirect', [\App\Http\Controllers\Auth\SocialiteController::class, 'redirect'])->name('redirect');
-    Route::match(['get', 'post'], '{provider}/callback', [\App\Http\Controllers\Auth\SocialiteController::class, 'callback'])->name('callback');
-    Route::get('{provider}/metadata', [\App\Http\Controllers\Auth\SocialiteController::class, 'metadata'])->name('metadata');
+    Route::post('{provider}/redirect', [Auth\SocialiteController::class, 'redirect'])->name('redirect');
+    Route::match(['get', 'post'], '{provider}/callback', [Auth\SocialiteController::class, 'callback'])->name('callback');
+    Route::get('{provider}/metadata', [Auth\SocialiteController::class, 'metadata'])->name('metadata');
 });
 
-Route::get('graph/{path?}', 'GraphController')
+Route::get('graph/{path?}', GraphController::class)
     ->where('path', '.*')
     ->middleware(['web', \App\Http\Middleware\AuthenticateGraph::class])->name('graph');
 
 // WebUI
-Route::group(['middleware' => ['auth'], 'guard' => 'auth'], function () {
```

Comment:
@murrant had to remove the `'guard' => 'auth'` for it to work, not sure if this is needed, there is no guard named auth?

Reply:
This is left to check